src/integration/telegram/telegram_bot.py

```
@@ -0,0 +1,338 @@
+            **Need support?** Contact your DELA administrator.
+            """
+            
+            await message.reply_text(help_message, parse_mode='Markdown')
+            
+        except Exception as e:
+            logger.error(f"Error in help command: {e}")
+            try:
+                message = update.message or update.effective_message
+                if message:
+                    await message.reply_text("Error showing help. Please try again.")
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)